spec/services/username_changer_spec.rb

```diff
@@ -335,7 +335,7 @@ def create_post_and_change_username(args = {}, &block)
               Lorem ipsum
 
               [quote="foo, post:1, topic:#{quoted_post.topic.id}"]
-              quoted post
```

Comment:
Hmm why was this spec changed?

Reply:
It is a full quote of the previous post, which was unnecessary and thus will be removed.